daemon/daemon.go

```diff
@@ -266,10 +266,16 @@ func Start(sockFile string, c *cli.Context) error {
 	config := &daemonConfig{
 		Root: root,
 	}
-	exists, err := util.ObjectExists(config)
-	if err != nil {
-		return err
+
+	ignoreCfgFile := c.Bool("ignore-config-file")
```

Comment:
I understand this part. My concern is about use cases. You know, besides the convoy.cfg, there are also configure files for each driver, under e.g. convoy/vfs/ directory, as well as volume metadata files. How should we handle them? Ignore the previous driver parameters(and volumes) totally? Or doesn't allow modify driver and driver parameters? Or handle it in some other way? I am not very sure of this if we want to introduce this option.